lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -308,6 +358,7 @@ def create_network(self):
                                                   driver=self.parameters.driver,
                                                   options=self.parameters.driver_options,
                                                   ipam=ipam_config,
+                                                  enable_ipv6=self.parameters.enable_ipv6,
```

Comment:
The `enable_ipv6` option was added in 1.10.0 of docker-py (see [changelog](https://docker-py.readthedocs.io/en/stable/change-log.html)). You'll need to either conditionally enable this parameter (see `docker-container` for examples of how to do this), or update the minimum version of docker-py required (in both the documentation above and the `AnsibleDockerClient` initialization below).

Reply:
Is it acceptable to set requirements for the whole docker_network to python >= 2.7 and docker-py >= 2.0.0 ? For me, this is the preferred method.

Reply:
No, I don't think that's a good idea.